proto/sifnode/ethbridge/v1/tx.proto

```diff
@@ -46,17 +48,33 @@ message MsgBurnRequest {
 
 message MsgBurnResponse {}
 
-message MsgCreateEthBridgeClaimRequest {
+message MsgCreateEthBridgeClaim {
   EthBridgeClaim eth_bridge_claim = 1 [(gogoproto.moretags) = "yaml:\"eth_bridge_claim\""];
 }
 
 message MsgCreateEthBridgeClaimResponse {}
 
 // MsgUpdateWhiteListValidator add or remove validator from whitelist
-message MsgUpdateWhiteListValidatorRequest {
+message MsgUpdateWhiteListValidator {
   string cosmos_sender  = 1 [(gogoproto.moretags) = "yaml:\"cosmos_sender\""];
   string validator      = 2 [(gogoproto.moretags) = "yaml:\"validator\""];
   string operation_type = 3 [(gogoproto.moretags) = "yaml:\"operation_type\""];
```

Comment:
Both `cosmos_sender` and `validator` should be an sdk.address type, not a string.